samples/cpp/ufacedetect.cpp

```diff
@@ -20,13 +20,13 @@ static void help()
             "This classifier can recognize many kinds of rigid objects, once the appropriate classifier is trained.\n"
             "It's most known use is for faces.\n"
             "Usage:\n"
-            "./facedetect [--cascade=<cascade_path> this is the primary trained classifier such as frontal face]\n"
+            "./ufacedetect [--cascade=<cascade_path> this is the primary trained classifier such as frontal face]\n"
```

Comment:
Hmm okay, so it uses the T-API... Not sure if we should combine them then. @mshabunin what do you think?